services/generic/register/register.go

```diff
@@ -0,0 +1,8 @@
+// Package register registers the generic service
```

Comment:
also need to register in services/register/all.go

Reply:
Ooh good catch; didn't know about that top level registration.